### DISCUSSION TITLE
终始全屏建议不要强制  可以用手动选择的方式

### DISCUSSION THREAD

**Discussion by fanwei001** [eiyooooo/Easycontrol_For_Car#21] [General]
有的车机安装后 没有提示界面 自动终始全屏，体验不好，进去后又关不了，还有大神，画中画版本有什么区别呀

**Comment by eiyooooo** [comment]
画中画版本是给某些车机有画中画功能使用的，画中画版本只有全屏模式故没有始终全屏模式开关，并且画中画版本包名和悬浮版有区别，可以共存

悬浮版若检测到没有悬浮窗权限才会提示授权“悬浮窗”或打开“始终全屏模式”，没有提示则默认悬浮、始终全屏模式关闭

悬浮版若有画中画需要则需打开“设置-其它-始终全屏模式”，若在画中画中使用且没打开“始终全屏模式”则有可能画面出错
